examples/hlsl/default.hlsl

```diff
@@ -0,0 +1,22 @@
+// The entry point and target profile are needed to compile this example:
+// -T ps_6_6 -E PSMain
```

Comment:
This is cool for now. (We need a more general solution for the "we'd like to provide arguments but allow users to override them" type of thing. We've never found a good solution on our side.)

Reply:
thanks!

Reply:
Yeah, it is similar to https://github.com/compiler-explorer/compiler-explorer/issues/3765.